app/src/main/java/comcmput301f16t01/github/carrier/RequestController.java

```diff
@@ -48,25 +40,17 @@ public String addRequest(Request request) {
         } else {
             ElasticRequestController.AddRequestTask art = new ElasticRequestController.AddRequestTask();
             art.execute(request);
-
             return null;
         }
     }
 
-    /**
-     * @param rider
-     * @return
-     */
-    public ArrayList<Request> getRequests(User rider) {
-        return requestList;
-    }
-
-    /**
-     *
-     */
-    public void reset() {
+    /** Clears information in the singleton, not exactly necessary */
+    // TODO check the necessity of this function.
+    public void clear() {
+        requestList = new RequestList();
     }
 
+    // TODO Why does this need a rider? You can cancel a request just knowing the request.
```

Comment:
Agreed. A driver can also cancel a request, can they not? So we don't care who is doing it.